release.json

```diff
@@ -45,12 +45,12 @@
     },
     "release-a6": {
         "INTEGRATIONS_CORE_VERSION": "7.56.0-rc.2",
-        "OMNIBUS_SOFTWARE_VERSION": "7.56.0-rc.1",
+        "OMNIBUS_SOFTWARE_VERSION": "7.56.0-rc.3",
```

Comment:
I just noticed this is `7` in the a6 key, is that intentional?

Reply:
Yea because we don't create `6.XX.0-rc.Y` tags on omnibus_software only macos repo has a separate agent 6 and agent 7 tag